README.md

```diff
@@ -273,35 +265,27 @@ lpm.onDisconnect(function(expected) {
 });
 ```
 
-### isConnected() ### 
-
-Returns the device's connectivity status. This is the same as calling Connection Manager's isConnected method.
+### isConnected() ###
 
-#### Parameters ####
-
-None.
+This method indicates whether the is connected or not. This is equivalent to calling Connection Manager’s *isConnected()* method.
 
 #### Return Value ####
 
-Boolean, `true` if the device is connected and `false` otherwise.
+Boolean &mdash; `true` if the device is connected, otherwise `false`.
 
 #### Example ####
 
 ```squirrel
-if (lpm.isConnected()) server.log("Device is connected");
+server.log("Device is " + (lpm.isConnected() ? "connected" : "disconnected"));
```

Comment:
The device should not log when disconnected. We should not be recommending this practice in examples.